lego/apps/events/views.py

```diff
@@ -203,6 +205,30 @@ def perform_update(self, serializer):
         """
         serializer.save(is_ready=False)
 
+    @decorators.action(detail=True, methods=["GET"])
+    def csv(self, *args, **kwargs):
+        event = Event.objects.get(pk=kwargs["pk"])
+        response = HttpResponse(content_type="text/csv")
+        response["Content-Disposition"] = (
+            f'attachment; filename="{event.title.replace(" ", "_") + "_attendees"}.csv"'
```

Comment:
Could you change this to `filename="{event.title.replace(" ", "_") }_attendees.csv"`